### PR TITLE
Export html to pdf

### PR DESCRIPTION
**Pull request by brandonwong368**
### Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/086c1b564dbf478faa649a8880e34b81?v=9d3f8002a0d1477ba395616dfc082af7&p=1be01de27e1c462bade88d67e612f917

### Implementation description

- using html-pdf package
- placeholder variable for local path to html file or html string to convert into pdf
- downloads file to a designated path 

### Steps to test

1. Added a random html file (`test.html` match the filename to the util) to the repository root 
2. Created a new endpoint calling the function. Used postman to send a request
3. New pdf called `test.pdf` will be downloaded (the test html file and the pdf were deleted after testing)

### What should reviewers focus on?

-

### Checklist

-   [ ] My PR name is descriptive and in imperative tense
-   [ ] I have run the linter
-   [ ] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by feng-eric** [comment]
@brandonwong368 Can you join the railway project if you haven't so we can do a PR deploy to be able to easily test this?

File: utils/htmlToPDF.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { create, CreateOptions } from "html-pdf";
+import { readFileSync } from "fs";
+
+/**
+ * converts hard coded html into a PDF and downloads the PDF locally
+ * @param {string} downloadPath download path of the generated pdf
+ */
+const htmlToPDF = (downloadPath: string): void => {
+    // path of the hard coded html or the html as a string
+    const htmlPath = "./test.html";
```
Comment (mahadzaryab1) [inline utils/htmlToPDF.ts:10]: @brandonwong368 @feng-eric how would we do this in a real use-case? are we going to have an HTML string within our code?

**Review by mahadzaryab1 — COMMENTED** [review]
added a question

File: utils/htmlToPDF.ts (hunk 1 above)
Comment (feng-eric) [reply utils/htmlToPDF.ts:10]: Yeah, we'll probably store a hard coded HTML string and fill it in with variables for user specific data

**Pull request closed without merging**